packages/rum-core/src/browser/htmlDomUtils.spec.ts

```diff
@@ -61,10 +61,16 @@ if (!isIE()) {
   describe('isShadowRoot', () => {
     const parent = document.createElement('div')
     parent.attachShadow({ mode: 'open' })
+    const form = document.createElement('form')
+    const input = document.createElement('input')
+    input.setAttribute('name', 'host')
+    form.appendChild(input)
     const parameters: Array<[Node, boolean]> = [
       [parent.shadowRoot!, true],
+      [form, false],
       [parent, false],
       [document.body, false],
+      [input, false],
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** not sure to get why we should add a check on this one, ‏what was your intent there?

Reply:
I added mostly because it was easy. I feel we should add test on a bunch of tags. For example `<a href="http://localhost/some/place.html"></a>` have a `host` property but it's not an element therefore it was not failing.
I think we would get a lot of value from property testing, were there discussion in your team about adding it?

Reply:
Indeed, that sounds like a nice test case to add!
It could definitely be interesting to have a test case for all possible combination of `has host attribute`, `is doc fragment` and `is element`. 

IMO, the current parametrised strategy makes a bit hard to understand what exactly we want to test for each case, it may be interesting to move to dedicated tests to better express what is interesting about each case. 

About property based testing, I don't think we had chat about it but that could definitely be something to explore at some point 👍